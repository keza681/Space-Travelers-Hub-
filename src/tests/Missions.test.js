import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../Redux/ConfigureStore';
import MissionList from '../components/HomeRocket/HomeRocketList';

describe('Mission component', () => {
  it('Should returns a row', () => {
    const app = renderer
      .create(
        <Provider store={store}>
          <MissionList />
        </Provider>,
      )
      .toJSON();
    expect(app).toMatchSnapshot();
  });
});
