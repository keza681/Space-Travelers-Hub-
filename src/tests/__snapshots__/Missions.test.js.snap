// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mission component Should returns a row 1`] = `
<div
  className="rockets"
/>
`;
